testsuite/unit-tests/src/test/java/org/jboss/resteasy/test/mapper/ExceptionHandlerTest.java

```diff
@@ -0,0 +1,170 @@
+package org.jboss.resteasy.test.mapper;
+
+import org.jboss.resteasy.core.ExceptionHandler;
+import org.jboss.resteasy.mock.MockHttpRequest;
+import org.jboss.resteasy.spi.ApplicationException;
+import org.jboss.resteasy.spi.HttpRequest;
+import org.jboss.resteasy.spi.HttpResponseCodes;
+import org.jboss.resteasy.spi.LoggableFailure;
+import org.jboss.resteasy.spi.NoLogWebApplicationException;
+import org.jboss.resteasy.spi.ResteasyProviderFactory;
+import org.jboss.resteasy.spi.UnhandledException;
+import org.jboss.resteasy.test.mapper.resource.ApplicationExceptionMapper;
+import org.jboss.resteasy.test.mapper.resource.IOExceptionMapper;
+import org.jboss.resteasy.test.mapper.resource.SprocketDBException;
+import org.jboss.resteasy.test.mapper.resource.SprocketDBExceptionMapper;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Test that the Exception processing and ExceptionMapper handling rules defined in
+ * the jar-311 section 3.3.4 are dealt with correctly.  Related information is provided
+ * in the Resteasy guide section 3.3.4
```

Comment:
I assume you're referring to section 30 of the doc, http://docs.jboss.org/resteasy/docs/3.6.1.Final/userguide/html_single/index.html#ExceptionHandling , right?
And please turn "jar-311" to "JSR-311"

Reply:
Its seciton 29 in mater's docbook, resteasy-reference-guide-en-US.pdf

Fixed